HappyAnding/HappyAnding/Model/ShortcutGrade.swift

```diff
@@ -14,4 +14,58 @@ enum ShortcutGrade: Int {
     case level10 = 3
     case level25 = 4
     case level50 = 5
```

Comment:
```suggestion
    case level3 = 3
    case level4 = 4
    case level5 = 5
```

해당 부분은 제가 전에 작업한 부분인데 레벨 네이밍이 1,2,3,4,5 로 확정되면서 같이 맞춰주신다면 더 좋을 것 같습니다~! 위의부분까지 제안이 가능하지 않아서 가능한 부분까지만 해두겠습니다

Reply:
변수 통일은 항상!!대 찬 성 입니다.

다만 저희 레벨 네이밍 있어야.. 되겠죠 ? ? ? ? ?

Reply:
저는 Level 0 ~Level 5 네이밍도 best는 아니어도 나쁜 네이밍은 아니라고 생각합니당! 다른 분들 의견도 더 듣고싶어요 👀

Reply:
변수명은 0~5로 통일했습니다! 에셋명은 유지할게요